Makefile

```diff
@@ -43,6 +44,7 @@ NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
 NPM ?= ./deps/npm/bin/npm-cli.js
+TEST_CI = ./tools/test-ci.sh
```

Comment:
Or hooking it into `git` (since it's already a requirement, and is platform uniform), then we could do
```console
$ git tool test-cl parallel/test-stream2-transform
$ git tool lint
$ git tool benchmark buffers/buffer-tostring.js len=1024
```